frontend/app/components/import_list/import.component.ts

```diff
@@ -36,10 +41,48 @@ export class ImportComponent implements OnInit {
     this.onImportList();
   }
 
+  ngAfterViewInit(): void {
+    // Called after ngAfterContentInit when the component's view has been initialized. Applies to components only.
+    // Add 'implements AfterViewInit' to the class.
+    fromEvent(this.search.nativeElement, 'keydown')
```

Comment:
```suggestion
    this.search.valueChanges.subscribe(value => {
    
    })
```
Cela évite aussi de devoir attendre que la vue soit initiée et de tout mettre dans le `ngOnInit()`